pages/SendScreen.tsx

```diff
@@ -41,7 +41,7 @@ export function SendScreen() {
   const [isLoading, setLoading] = useState(true); // Loading status => only show the responce of the API
 
   // after the request completes
-  const [data, setData] = useState(""); // Dynamically loaded data from the Interclip REST API
+  const [data, setData] = useState({ result: "" }); // Dynamically loaded data from the Interclip REST API
```

Comment:
@filiptronicek Missed to mention what her type is :D

Reply:
Poor little useState :(